consensus/src/epoch_manager.rs

```diff
@@ -1032,11 +1038,12 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
                 epoch_state,
                 consensus_config,
                 execution_config,
+                onchain_randomness_config,
+                jwk_consensus_config,
```

Comment:
why it needs to understand jwk config?

Reply:
ircc that's for round manager needs to know whether the jwk txn in proposal is expected